library/Icingadb/Widget/Detail/UsergroupDetail.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+/* Icinga DB Web | (c) 2021 Icinga GmbH | GPLv2 */
+
+namespace Icinga\Module\Icingadb\Widget\Detail;
+
+use Icinga\Module\Icingadb\Common\Auth;
+use Icinga\Module\Icingadb\Model\Usergroup;
+use Icinga\Module\Icingadb\Widget\EmptyState;
+use Icinga\Module\Icingadb\Widget\ItemList\UserList;
+use ipl\Html\BaseHtmlElement;
+use ipl\Html\HtmlElement;
+use ipl\Html\Text;
+use ipl\Web\Url;
+use ipl\Web\Widget\Link;
+
+class UsergroupDetail extends BaseHtmlElement
+{
+    use Auth;
+
+    /** @var int The amount of users in the list */
+    const LIST_LIMIT = 6;
+
+    /** @var Usergroup The given user group */
+    protected $usergroup;
+
+    protected $defaultAttributes = ['class' => 'usergroup-detail'];
+
+    protected $tag = 'div';
+
+    public function __construct($usergroup)
+    {
+        $this->usergroup = $usergroup;
+    }
+
+    protected function createCustomVars()
+    {
+        $content = [new HtmlElement('h2', null, Text::create(t('Custom Variables')))];
+        $flattenedVars = $this->usergroup->customvar_flat;
+        $this->applyRestrictions($flattenedVars);
+
+        $vars = $this->usergroup->customvar_flat->getModel()->unflattenVars($flattenedVars);
+        if (! empty($vars)) {
+            $customvarTable = new CustomVarTable($vars);
+            $customvarTable->setAttribute('id', 'usergroup-customvars');
+            $content[] = $customvarTable;
+        } else {
+            $content[] = new EmptyState(t('No custom variables configured.'));
+        }
+
+        return $content;
+    }
+
+    protected function createUserList()
+    {
+        $showMoreLink = null;
+        $userList = $this->usergroup->user;
+        $url = Url::fromRequest();
+
+        if (count($userList) > self::LIST_LIMIT && ! $url->hasParam('fullList')) {
+            foreach ($userList as $key => $user) {
+                if ($key === self::LIST_LIMIT) {
+                    $showMoreLink = (new Link(
+                        'Show more',
+                        $url->setParam('fullList')
+                    ));
+
+                    break;
+                }
+                    $list[$key] = $user;
+            }
+
+            $userList = $list;
+        }
```

Comment:
Have you taken a look at how the *show more* link is setup elsewhere? (Hint: There's a `ShowMore` widget) What you're doing here is .. interesting :joy: